rest-server/src/models/job.js

```diff
@@ -277,13 +277,15 @@ class Job {
   }
 
   generateFrameworkDescription(data) {
+    const gpuType = data.gpuType || null;
```

Comment:
Does this line mean that gpuType is null when data.gpuType is ‘’?

Reply:
yes, it's the format required by launcher